apps/meteor/tests/end-to-end/api/01-users.js

```diff
@@ -4017,8 +3500,8 @@ describe('[Users]', function () {
 		});
 
 		describe('[with permission]', () => {
-			before(() => {
-				updatePermission('view-outside-room', ['admin', 'user']);
+			before(async () => {
```

Comment:
it should work if you just remove the `{`, so you won't need the `async` and the `await`, just return the promise